scripts/check-payload/config.toml

```diff
@@ -234,12 +234,20 @@ files = [
 
 # https://issues.redhat.com/browse/RHOAIENG-24290
 
+# when scanning with `scan image --spec`
 [[payload.python-311-container.ignore]]
 error = "ErrNotDynLinked"
 files = [
     # executable is not dynamically linked
     "/opt/app-root/bin/py-spy",
 ]
+# when scanning with `scan local --path`, the above does not apply
```

Comment:
So - is the above still needed since we're not running it that way anymore?

Reply:
Okay then, I wouldn't know how to run it locally myself - maybe we could have some target in the Makefile? anyway, that would be a different PR.

Reply:
* https://github.com/opendatahub-io/notebooks/issues/1086